src/libcommonserver/utility/utility.cpp

```diff
@@ -452,6 +452,10 @@ std::string Utility::opType2Str(OperationType opType) {
     }
 }
 
+std::wstring Utility::opType2WStr(OperationType opType) {
```

Comment:
Raising a topic that is beyond the scope of this PR: why not using converters whose name is simply `toStr` and `toWStr`? It would be easier to remember and much less to type.